modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/playback/MediaSessionManager.kt

```diff
@@ -112,6 +113,17 @@ class MediaSessionManager(
             .addTo(disposables)
     }
 
+    private fun observeHideCustomMediaActions() {
+        launch {
+            settings.maxCustomMediaActionsCountFlow.collect {
+                withContext(Dispatchers.Main) {
+                    val playbackStateCompat = getPlaybackStateCompat(playbackManager.playbackStateRelay.blockingFirst(), currentEpisode = playbackManager.getCurrentEpisode())
+                    updatePlaybackState(playbackStateCompat)
+                }
+            }
```

Comment:
This is just listening to the flow in order to recreate the playback state anytime the flow updates, right? If so, it might be worth adding a comment here clarifying that it is just listening for changes (but not actually using updated values). Just a thought I had since I had to read this chunk of code a few times before I realized I wasn't just overlooking where the updated value was being used. I don't feel strongly.

Reply:
Added a comment in e8e7d1e